litex/soc/cores/spi_xip/phy/generic.py

```diff
@@ -0,0 +1,122 @@
+from migen import *
+from migen.genlib.fsm import FSM, NextState
+
+from litex.soc.cores.spi_xip.common import *
+from litex.soc.cores.spi_xip.clkgen import *
+
+from litex.soc.interconnect import stream
+
+cmd_oe_mask = 0b0001
+soft_oe_mask = 0b0001
+addr_oe_mask = {
+    1: 0b0001,
+    2: 0b0011,
+    4: 0b1111,
+}
+
+def GetConfig(flash=None):
+    if flash is None:
+        return (24, 8, 1, 1, 1, 0x0b)
```

Comment:
Magic numbers - what do these mean? Should it be returning a named tuple or some other similar structure?